test/compilable/imports/imp16085b.d

```diff
@@ -0,0 +1,17 @@
+import imp16085 : S;
+
```

Comment:
I won't get into this, the current Alias  implementation is hopelessly broken and needs to be replaced.
```d
int a() { return 0; }
alias a = x => 1; // works
alias b = x => 2; // bug
int b() { return 3; }

unittest
{
    assert(a() == 0);
    assert(a(1) == 1);
    assert(b(1) == 2);
    assert(b() == 3);
}
```